app/notification/routes.py

```diff
@@ -21,7 +23,7 @@
 
 
 @notification_bp.route("", methods=["POST"])
-def send_notification() -> dict:
+def send_email() -> dict:
```

Comment:
🤔 Does it return a dict? Or a Response?

Reply:
dict response

Reply:
I think @SentientHousePlant has a point here. If I run it through `mypy` I get:
```
app/notification/routes.py:44: error: Incompatible return value type (got "Response", expected "Dict[Any, Any]")
```

Reply:
it expects Dict(response). Although happy to remove it

Reply:
It returns a response - so you just need to change "-> dict" to "-> Response" 😄